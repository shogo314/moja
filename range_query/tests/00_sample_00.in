3
100 150 3
0
10 -120 -1
3
3
8 2 3
4
8
