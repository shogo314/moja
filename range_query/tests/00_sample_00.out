2108
7
0
